### DISCUSSION TITLE
6.0.0 forcing to buy a new Mac?

### DISCUSSION THREAD

**Discussion by echogit** [bluefireteam/audioplayers#1794] [General]
After spending a few hours trying to get 6.0.0 to work on my Mac iOS simulator, ended up finding out that it will only work on XCode 15, which means it won't work on the only Mac I have :(

The error is:
`
Failed to build iOS app
Swift Compiler Error (Xcode): Cannot find 'AVURLAssetOverrideMIMETypeKey' in scope
WrappedMediaPlayer.swift:184:36
`

I checked I can use OpenCore to install a new version of the MacOS (ventura) so I can use the newer versions of XCode, but not sure if that will work and until I can, I'd like to not have to buy a new Mac.

For now the solution was to downgrade audioplayers to 5.2.0

**Comment by Gustl22** [comment]
Unfortunately that's the case, but only for building. The application itself should also run on older software, once build. In order to make some features work on newer iOS / macOS versions, we were forced to do so. 

We are not the ones to blame here. Blame Apple. If it's about documentation, we can drop something into the Troubleshooting guide.

**Comment by echogit** [comment]
I ended up using OpenCore Legacy Patcher, and that did the job
